test/utils/input/utils.js

```diff
@@ -0,0 +1,426 @@
+			inquirer.prompt.resolves(promptResult);
+			const passphrase = await inputUtils.getPassphraseFromPrompt({
+				displayName,
+			});
+			return expect(passphrase).to.equal(promptResult.passphrase);
+		});
+
+		it('should prompt once with shouldRepeat false', async () => {
+			const promptResult = { passphrase: '123' };
+			inquirer.prompt.resolves(promptResult);
+			await inputUtils.getPassphraseFromPrompt({
+				displayName,
+			});
+			return expect(inquirer.prompt).to.be.calledWithExactly([
+				{
+					name: 'passphrase',
+					type: 'password',
+					message: `Please enter ${displayName}: `,
+				},
+			]);
+		});
+
+		it('should prompt twice with shouldRepeat true', async () => {
+			const promptResult = { passphrase: '123', passphraseRepeat: '123' };
+			inquirer.prompt.resolves(promptResult);
+			await inputUtils.getPassphraseFromPrompt({
+				shouldRepeat: true,
+				displayName,
+			});
+			return expect(inquirer.prompt).to.be.calledWithExactly([
+				{
+					name: 'passphrase',
+					type: 'password',
+					message: `Please enter ${displayName}: `,
+				},
+				{
+					name: 'passphraseRepeat',
+					type: 'password',
+					message: `Please re-enter ${displayName}: `,
+				},
+			]);
+		});
+
+		it('should reject with error when repeated passphrase does not match', () => {
+			const promptResult = { passphrase: '123', passphraseRepeat: '456' };
+			inquirer.prompt.resolves(promptResult);
+			return expect(
+				inputUtils.getPassphraseFromPrompt({ shouldRepeat: true, displayName }),
+			).to.be.rejectedWith(
+				ValidationError,
+				'Password was not successfully repeated.',
+			);
+		});
+	});
+
+	describe('#getPassphraseFromEnvVariable', () => {
+		const displayName = 'passphrase';
+		beforeEach(() => {
+			delete process.env.PASSPHRASE;
+			return Promise.resolve();
+		});
+
+		after(() => {
+			delete process.env.PASSPHRASE;
+			return Promise.resolve();
+		});
+
+		it('should reject with validation error when passphrase does not exist', () => {
+			return expect(
+				inputUtils.getPassphraseFromEnvVariable('PASSPHRASE', displayName),
+			).to.be.rejectedWith(
+				ValidationError,
+				`Environmental variable for ${displayName} not set.`,
+			);
+		});
+
+		it('should resolve with the set passphrase', () => {
+			const passphrase = 'somepassphrase';
+			process.env.PASSPHRASE = passphrase;
+			return expect(
+				inputUtils.getPassphraseFromEnvVariable('PASSPHRASE', displayName),
+			).to.eventually.equal(passphrase);
+		});
+	});
+
+	describe('#getPassphraseFromFile', () => {
+		const filePath = '/path/to/the/passphrase.txt';
+
+		describe('when file does not exist', () => {
+			beforeEach(() => {
+				const error = new Error('ENOENT: no such file or directory');
+				const streamStub = createStreamStub(
+					(type, callback) => type === 'error' && callback(error),
+				);
+				return sandbox.stub(fs, 'createReadStream').returns(streamStub);
+			});
+
+			it('should throw an error when file does not exist', () => {
```

Comment:
`when file does not exist` is already covered by the enclosing suite description.